examples/pytorch/pointcloud/pointnet/README.md

```diff
@@ -7,13 +7,21 @@ This is a reproduction of the papers
 
 # Performance
 
+## Classification
```

Comment:
Also, if per-epoch training time numbers are available for ours and the reference implementation, you can also put into the table.

Reply:
Not sure whether to mention this if it is machine-dependent

Reply:
You can also add your environment config (CPU model, GPU model, RAM size, etc).